CONTRIBUTING.md

```diff
@@ -155,3 +156,28 @@ To build and run the android app (emulator or real android device is needed)
 ```shell
 $ ./gradlew :app-android:installDebug
 ```
+
+### 🔬 Verifications
+
+To verify important checks, run the following commands on your local machine
+
+#### General Check
+
+Run all configured checks for the project
+```shell
+$ ./gradlew check
+```
```

Comment:
what all does this run? just unit tests?

Reply:
The check task should run all tests and detekt (once allowed)